### ISSUE TITLE
More than 100 mb file is not uoloading

### ISSUE THREAD

**Issue by connectmadan** [blueimp/jQuery-File-Upload#2547]
Hi,

This plugin is not uploading more than 100 mb file as it says that file is uploaded but actually file is not uploading, How will i handle if file is not uploaded

The responce message is howing as follow

<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>

<script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.1/jquery-ui.min.js"></script>

<script src="js/tmpl.min.js"></script>

<script src="js/load-image.min.js"></script>

<script src="js/canvas-to-blob.min.js"></script>

<script src="js/jquery.image-gallery.min.js"></script>

<script src="js/jquery.iframe-transport.js"></script>

<script src="js/jquery.fileupload.js"></script>

<script src="js/jquery.fileupload-fp.js"></script>

<script src="js/jquery.fileupload-ui.js"></script>

<script src="js/jquery.fileupload-jui.js"></script>

<script src="js/main.js"></script>

<script id="template-upload" type="text/x-tmpl">
{% for (var i=0, file; file=o.files[i]; i++) { %}
    <tr class="template-upload fade">
        <td class="preview"><span class="fade"></span></td>
        <td class="name"><span>{%=file.name%}</span></td>
        <td class="size"><span>{%=o.formatFileSize(file.size)%}</span></td>
        {% if (file.error) { %}
            <td class="error" colspan="2"><span class="label label-important">Error</span> {%=file.error%}</td>
        {% } else if (o.files.valid && !i) { %}
            <td>
                <div class="progress progress-success progress-striped active" role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow="0"><div class="bar" style="width:0%;"></div></div>
            </td>
            <td>{% if (!o.options.autoUpload) { %}
                <button class="btn btn-primary button upload start">
                    <i class="icon-upload icon-white"></i>
                    <span>Start</span>
                </button>
            {% } %}</td>
        {% } else { %}
            <td colspan="2"></td>
        {% } %}
        <td>{% if (!i) { %}
            <button class="btn btn-warning button orange cancel">
                <i class="icon-ban-circle icon-white"></i>
                <span>Cancel</span>
            </button>
        {% } %}</td>
    </tr>
{% } %}
</script>

<script id="template-download" type="text/x-tmpl">
{% for (var i=0, file; file=o.files[i]; i++) { alert(file.error); %}
    <tr class="template-download fade">
            <td class="preview">{% if (file.thumbnail_url) { %}
            {% } %}</td>
            <td class="name">{%=file.name%}</td>
            <td class="size"><span>{%=o.formatFileSize(file.size)%}</span></td>
            <td colspan="2"></td>
        <td>Succesfully Uploaded!
        </td>
    </tr>
{% } %}
</script>
